paparazzi-gradle-plugin/src/test/projects/prepare-resources-task-caching/build.gradle.bak

```diff
@@ -0,0 +1,25 @@
+plugins {
```

Comment:
this file is repetitive, let's use the system property approach like `verify-update-aar-resources-change/build.gradle` or even better, let's update that test project instead adding a new one.

....unless there's a clear difference between a local aar and repository aar project?  If so, I'd still like the tests to clearly surface that difference, if possible.